drivers/ethernet/eth_nxp_imx_netc_priv.h

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_ETHERNET_ETH_NXP_IMX_NETC_PRIV_H_
+#define ZEPHYR_DRIVERS_ETHERNET_ETH_NXP_IMX_NETC_PRIV_H_
+
+#include "fsl_netc_endpoint.h"
+#include "fsl_msgintr.h"
+
+/* Buffer and descriptor alignment */
+#define NETC_BD_ALIGN   128
+#define NETC_BUFF_ALIGN 64
+#define NETC_RX_RING_BUF_SIZE_ALIGN                                                                \
+	SDK_SIZEALIGN(CONFIG_ETH_NXP_IMX_RX_RING_BUF_SIZE, NETC_BUFF_ALIGN)
+
+/* Buffer descriptor ring */
+#define NETC_RING_NUM 3
```

Comment:
I'm assuming this is the number of rings on the hardware, should this be captured in DT (same comment for any other hardware related `#define`'s in this file

Reply:
This should be configuration of ring number to use, but not hardware description.
I think moving to Kconfig may be okay?

Reply:
you can make it a Kconfig but if you don't want to support configurability of that number right now you don't have to